crates/holochain/src/core/ribosome/host_fn/accept_countersigning_preflight_request.rs

```diff
@@ -158,6 +158,7 @@ pub mod wasm_test {
 
     #[tokio::test(flavor = "multi_thread")]
     #[cfg(feature = "slow_tests")]
+    #[cfg_attr(target_os = "macos", ignore = "flaky")]
```

Comment:
@steveej you could take this one.

Reply:
This is a timing issue. @steveej Are you working on this already? Otherwise I'd include the fix in my PR.

Reply:
@jost-s yes but i'm happy to review a fix on your PR and try if it fixes the error for me.

Reply:
Alright, this is it https://github.com/holochain/holochain/pull/3074/commits/f386998d45d3ce33e1a92f9b470de5f184ac9837

Reply:
confirming the fix, thanks!

> test core::ribosome::host_fn::accept_countersigning_preflight_request::wasm_test::unlock_invalid_session ... ok